index.html

```diff
@@ -68,15 +68,23 @@ <h3>Settings</h3>
     <div class="buttons">
       <button id="resetBtn" class="action" onclick="resetDefaultSettings()">Restore defaults</button>
       <button id="submitBtn" class="action" onclick="submitSettings()">Apply</button>
+      <div id = 'github'>
+      <a href="https://github.com/zenika-open-source/clepsydre">
+        <p>Contribute to our project</p>
+        <img src="img/github-mark-white.png" alt="Github" />
+      </a>
```

Comment:
```suggestion
      <a href="https://github.com/zenika-open-source/clepsydre" title="contribute to the project on Github">
        Contribute on Github <img src="img/github-mark-white.png" alt="Github logo" />
      </a>
```

But you didn't use the svg file, instead you added 3 png to the project.